### PR TITLE
minFraud Node library has API documentation

### PR DESCRIPTION
**Pull request by kevcenteno**
Add documentation to response and request parameters

### CODE REVIEW

File: src/request/account.ts [hunk 1]
```diff
@@ -1,12 +1,36 @@
 import * as crypto from 'crypto';
 
 interface AccountProps {
+  /**
+   * A unique user ID associated with the end-user in your system. If your
+   * system allows the login name for the account to be changed, this should
+   * not be the login name for the account, but rather should be an internal ID
+   * that does not change. This is not your MaxMind user ID.
```
Comment (oschwald) [inline src/request/account.ts:8]:
```suggestion
   * that does not change. This is not your MaxMind account ID.
```

File: src/request/account.ts [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+/**
+ * Account related data information for the transaction being sent to the web
+ * service.
+ */
 export default class Account {
+  /**
+   * A unique user ID associated with the end-user in your system. If your
+   * system allows the login name for the account to be changed, this should
+   * not be the login name for the account, but rather should be an internal ID
+   * that does not change. This is not your MaxMind user ID.
```
Comment (oschwald) [inline src/request/account.ts:27]:
```suggestion
   * that does not change. This is not your MaxMind account ID.
```

File: src/request/credit-card.ts [hunk 3]
```diff
@@ -16,14 +45,46 @@ const issuerIdNumberRegex = /^[0-9]{6}$/;
...
 const tokenRegex = /^(?![0-9]{1,19}$)[\u0021-\u007E]{1,255}$/;
 
+/**
+ * The credit card information for the transaction being sent to the web service.
+ */
 export default class CreditCard implements CreditCardProps {
+  /**
+   * The issuer ID number for the credit card. This is the first 6 digits of
+   * the credit card number. It identifies the issuing bank.
+   */
```
Comment (oschwald) [inline src/request/credit-card.ts:55]: Is there a way to inherit this documentation so we don't need to have it twice?

File: src/response/records.ts [hunk 4]
```diff
@@ -19,92 +29,364 @@ export interface GeoIPCountry extends CountryRecord {
...
+   * be parsed or was not provided or if the IP address could not be
+   * geolocated, the property will be `null`.
+   */
   readonly isInIpCountry?: boolean;
 }
 
 export interface BillingAddress {
+  /**
+   * This property is `true` if the postal code provided with the address is in
+   * the city for the address.The property is `false` when the postal code is not
```
Comment (oschwald) [inline src/response/records.ts:262]:
```suggestion
   * the city for the address. The property is `false` when the postal code is not
```

File: src/response/records.ts [hunk 5]
```diff
@@ -19,92 +29,364 @@ export interface GeoIPCountry extends CountryRecord {
...
+  /**
+   * The distance in kilometers from the address to the IP location.
+   */
   readonly distanceToIpLocation: number;
+  /**
+   * The distance in kilometers from the shipping address to billing address.
+   */
   readonly distanceToBillingAddress: number;
+  /**
+   * This property is `true` if the address is in the IP country.The property is
```
Comment (oschwald) [inline src/response/records.ts:251]:
```suggestion
   * This property is `true` if the address is in the IP country. The property is
```

File: src/response/records.ts [hunk 6]
```diff
@@ -19,92 +29,364 @@ export interface GeoIPCountry extends CountryRecord {
...
+  /**
+   * The longitude associated with the address.
+   */
   readonly longitude: number;
+  /**
+   * The distance in kilometers from the address to the IP location.
+   */
   readonly distanceToIpLocation: number;
+  /**
+   * This property is `true` if the address is in the IP country.The property is
```
Comment (oschwald) [inline src/response/records.ts:280]:
```suggestion
   * This property is `true` if the address is in the IP country. The property is
```

File: src/response/records.ts [hunk 7]
```diff
@@ -19,92 +29,364 @@ export interface GeoIPCountry extends CountryRecord {
...
+   */
   readonly billingAddressDistanceToIpLocation?: number;
+  /**
+   * The risk associated with the browser attributes such as the User-Agent and
+   * Accept-Language. If present, this is a value in the range 0.01 to 99.
+   */
   readonly browser?: number;
+  /**
+   * Individualized risk of chargeback for the given IP address on your account
+   * and shop ID.This is only available to users sending chargeback data to
```
Comment (oschwald) [inline src/response/records.ts:334]:
```suggestion
   * and shop ID. This is only available to users sending chargeback data to
```

File: src/response/records.ts [hunk 8]
```diff
@@ -19,92 +29,364 @@ export interface GeoIPCountry extends CountryRecord {
...
+  /**
+   * This property is `true` if the shipping address is in the IP country. The
+   * property is `false` when the address is not in the IP country. If the
+   * shipping address could not be parsed or was not provided or the IP address
+   * could not be geolocated, then the property is `null`.
+   */
   readonly isHighRisk: boolean;
+  /**
+   * This property is `true` if the postal code provided with the address is in
+   * the city for the address.The property is `false` when the postal code is not
```
Comment (oschwald) [inline src/response/records.ts:229]:
```suggestion
   * the city for the address. The property is `false` when the postal code is not
```

File: src/request/credit-card.ts (hunk 3 above)
Comment (kevcenteno) [reply src/request/credit-card.ts:55]: Not that I'm aware of.

Comment (oschwald) [reply src/request/credit-card.ts:55]: I think `/** @inheritdoc */` on the property will do it based on my testing. There may be a better way.

Comment (kevcenteno) [reply src/request/credit-card.ts]: Nice!  I've made the change here: https://github.com/maxmind/minfraud-api-node/pull/239/commits/0cd150ae187e3a3366599ac4b2ebc6b49c33d631

**Pull request merged**